dbt_project.yml

```diff
@@ -26,29 +27,74 @@ vars:
   partb_physicians: "{{ source('cclf','partb_physicians') }}"
   partb_dme: "{{ source('cclf','partb_dme') }}"
 
-# These configurations specify where dbt should look for different types of files.
-# The `source-paths` config, for example, states that models in this project can be
-# found in the "models/" directory. You probably won't need to change these!
+
+
+
+#### Output configurations
+### Database configuration
+  tuva_database:  tuva                        # Name of the database where cclf connector and ALL tuva packages will be written.
+  # medicare_cclf_connector_database: tuva    # Optional config that will apply to this project only, and take precedence over tuva_database.
+
+### Schema configuration
+## *Note*: without any schema configurations, this project will write to the defult schema, "claims_input"
```

Comment:
Maybe we should update the default schema to "claims_common" since we want to get away from the "claims input" term.

Reply:
done!